docs/usage/from-a-terminal.rst

```diff
@@ -741,11 +748,9 @@ Multiprocessing
 
 Sets the number of maximal worker to use.
 
-**Default value:** :code:`False`
+The default is number of available CPU cores multiplied by 5
```

Comment:
Actually, I changed that.

By default, if you have more than 2 CPUs it will be the `number of CPU - 2`. Otherwise `1`.

Reply:
:unamused: You know I just have rewritten that entirely :sob:

Reply:
Yes! And actually, everybody should test with their machine. PyFunceble mostly does IO Bound tasks so a higher number of workers is totally okay! But people should keep in mind that it depends on their machines. They should just test and look at loads of their machine 😄

Reply:
I thinks the translations are eating something here...

> By default, if you have more than 2 CPUs it will be the number of CPU - 2. Otherwise 1

That makes `-w` = `-p`  But `-w` IS != number of CPU cores/threads activated... is related to sub-processes...

> Yes! And actually, everybody should test with their machine

That makes the default of `-w`  = 5 :yum: 

Then #187 happens when I would look at the htop....
Leaving the comment here

Reply:
If `-w` = 4 then it spawn 6 sub processes of which only one open 6 other sub-processes...

![image](https://user-images.githubusercontent.com/44526987/104140632-ad32be00-53b2-11eb-95be-452aef858024.png)

Are you sure the worker/multi-processing is working as you thought? To me it seems weird to have 5 apparently passive processes.

Reply:
Moved this issue to <https://github.com/spirillen/PyFunceble/issues/34>

Closing as resolved for this PR to it's clear how it should be describet in a new MR

Reply:
That one should be moved to in progress https://github.com/funilrys/PyFunceble/projects/7#card-52531327